.eslintrc.json

```diff
@@ -67,7 +67,19 @@
           },
           {
             "selector": "variable",
-            "format": ["camelCase", "PascalCase", "UPPER_CASE"]
+            "format": [],
+            "custom": {
+              "regex": "^_([0-9])?$",
+              "match": true
+            }
+          },
+          {
+            "selector": "variable",
+            "format": ["camelCase", "PascalCase", "UPPER_CASE"],
+            "filter": {
+              "regex": "^_([0-9])?$",
+              "match": false
+            }
           },
```

Comment:
I don't understand what this change does. Can you rationalize it for me? What are you trying to do?

Reply:
We have an eslint rule called `@typescript-eslint/no-unused-vars` which throws an error of unused variables. We however sometimes need dummy variables in code:
```typescript
const [_, setFoo] = useState('')
```
So we make an exception by naming the variable as `_`. In some cases we need multiple dummy variables in the same scope, so we can suffix them with a number:
```typescript
const [_1, _2, height] = [2, 1, 5]
```

We recently introduced the `@typescript-eslint/naming-convention` eslint rule, which clashes with the `@typescript-eslint/no-unused-vars`, since we yet again (what this PR implements) have to make exceptions for `^_([0-9])?$`

Reply:
I would suggest removing this. Dummy variables can still have descriptive names even though they are not used. Using `_1`, `_2` is opaque and a naive solution to having to declare a variable. Much better, using your code, would be:
```js
const [_width, _depth, height] = [2, 1, 5]
```
This conveys meaning to future readers and makes no difference in code, since they aren't used.